Cannot access non-member variables in CEPuckEntity

I get the following errors when compiling against the latest version of ARGoS (3.0.0-beta48).

This can be fixed by hard-coding the variable values from `epuck_entity.cpp` into `dynamics2d_epuck_model.cpp`, though there may be a better solution.

```[31%] Building CXX object plugins/robots/e-puck/CMakeFiles/argos3plugin_simulator_epuck.dir/simulator/dynamics2d_epuck_model.cpp.o
/argos3-epuck/src/plugins/robots/e-puck/simulator/dynamics2d_epuck_model.cpp:17:69: error: no member named 'BODY_RADIUS' in 'argos::CEPuckEntity'
   static const Real EPUCK_RADIUS                   = CEPuckEntity::BODY_RADIUS;
                                                      ~~~~~~~~~~~~~~^
/argos3-epuck/src/plugins/robots/e-puck/simulator/dynamics2d_epuck_model.cpp:18:69: error: no member named 'INTERWHEEL_DISTANCE' in 'argos::CEPuckEntity'
   static const Real EPUCK_INTERWHEEL_DISTANCE      = CEPuckEntity::INTERWHEEL_DISTANCE;
                                                      ~~~~~~~~~~~~~~^
/argos3-epuck/src/plugins/robots/e-puck/simulator/dynamics2d_epuck_model.cpp:19:68: error: no member named 'BODY_HEIGHT' in 'argos::CEPuckEntity'
   static const Real EPUCK_HEIGHT                   =CEPuckEntity::BODY_HEIGHT;
                                                     ~~~~~~~~~~~~~~^
3 errors generated.
make[2]: *** [plugins/robots/e-puck/CMakeFiles/argos3plugin_simulator_epuck.dir/simulator/dynamics2d_epuck_model.cpp.o] Error 1
make[1]: *** [plugins/robots/e-puck/CMakeFiles/argos3plugin_simulator_epuck.dir/all] Error 2
make: *** [all] Error 2```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -37,8 +37,7 @@ include(${CMAKE_SOURCE_DIR}/cmake/ARGoSPackaging.cmake)
 
 
 # Set up ARGoS compilation information
-include_directories(${CMAKE_SOURCE_DIR} ${ARGOS_INCLUDE_DIRS})
-link_directories(${ARGOS_LIBRARY_DIRS})
+include_directories(BEFORE ${CMAKE_SOURCE_DIR})
 
 #
 # Compile stuff

